modules/extension/xsd/xsd-fes/src/main/java/org/geotools/filter/v2_0/FESConfiguration.java

```diff
@@ -18,9 +18,7 @@
 
 import net.opengis.fes20.Fes20Factory;
 import org.geotools.filter.FilterFactoryImpl;
-import org.geotools.filter.v1_1.SortByTypeBinding;
 import org.geotools.filter.v1_1.SortOrderTypeBinding;
```

Comment:
Is this still needed? and if it is should it be?

Reply:
The code is identical between V1 and V2, so I used the class v1_1.SortOrderTypeBinding. If you prefer, I can create an empty class  v2.SortOrderTypeBinding which extends v1_1.SortOrderTypeBinding

Reply:
No if they are identical then no problem, just caught my eye and have seen similar issues with this sort of mixing.